src/components/App/App.js

```diff
@@ -2,28 +2,38 @@ import React, { Component } from 'react';
 import movieData from '../../movieData';
 import Collection from '../Collection/Collection';
 import Nav from '../Nav/Nav';
+import MovieInfoView from '../MovieInfoView/MovieInfoView';
 
 import './App.css';
 
 class App extends Component {
   constructor() {
     super();
     this.state = {
-      movies:  movieData.movies
+      movies:  movieData.movies,
+      collectionView: true,
+      currentMovie: null
```

Comment:
We spoke in the audit this morning about using the empty data shape rather than `null` for your initial state value.   You may have already refactored this to avoid using null but if not, I'd consider it.